src/components/track-changes/suggestion-list/SuggestionSnippet.tsx

```diff
@@ -42,115 +50,178 @@ export const SuggestionSnippet: React.FC<{ suggestion: TrackedChange }> = ({
+              },
+            }
           }
-        } else if (suggestion.node.type === schema.nodes.footnote) {
-          return {
-            snippet: {
-              operation: changeOperationAlias(dataTracked.operation),
-              nodeName: suggestion.node.type.name,
-              content: getFootnoteText(view.state, suggestion.node),
-            },
-            message: '',
+          case schema.nodes.inline_equation:
+          case schema.nodes.equation_element: {
+            return {
+              snippet: {
+                operation: changeOperationAlias(dataTracked.operation),
+                nodeName: suggestion.node.type.spec.name,
+                content: getEquationContent(suggestion.node),
+                isEquation: true,
+              },
+            }
           }
-        } else {
-          return {
-            snippet: null,
-            message: `${suggestion.node.type.name
-              .charAt(0)
-              .toUpperCase()}${suggestion.node.type.name.slice(1)} ${
-              dataTracked.operation
-            }`,
+          case schema.nodes.section: {
+            const nodeName =
+              suggestion.node.attrs.category === 'MPSectionCategory:subsection'
+                ? 'Subsection'
+                : 'Section'
+            return {
+              snippet: {
+                operation: changeOperationAlias(dataTracked.operation),
+                nodeName: nodeName,
+                content: getFirstChildContent(suggestion.node),
+              },
+            }
           }
-        }
-      }
-
-      if (ChangeSet.isNodeAttrChange(suggestion)) {
-        if (suggestion.node.type === schema.nodes.inline_footnote) {
-          return {
-            snippet: {
-              operation: changeOperationAlias(dataTracked.operation),
-              nodeName: suggestion.node.type.spec.name,
-              content: getInlineFootnoteContent(doc, suggestion.newAttrs),
-            },
-            message: '',
-          }
-        } else if (suggestion.node.type === schema.nodes.footnote) {
-          return {
-            snippet: {
-              operation: changeOperationAlias(dataTracked.operation),
-              nodeName: suggestion.node.type.name,
-              content: getFootnoteText(view.state, suggestion.node),
-            },
-            message: '',
-          }
-        } else {
-          return {
-            snippet: null,
-            message: `${suggestion.node.type.name
-              .charAt(0)
-              .toUpperCase()}${suggestion.node.type.name.slice(1)} ${
-              dataTracked.operation
-            }`,
+          case schema.nodes.list: {
+            return {
+              snippet: {
+                operation: changeOperationAlias(dataTracked.operation),
+                nodeName: suggestion.node.type.spec.name,
+                content: `<span class="inspector_listItem">${getFirstChildContent(
```

Comment:
Please adjust this class name so it corresponds to our general approach. I think we just use dashes styles, so it'll be inspector-list-item.

Reply:
done